app/namespace.go

```diff
@@ -1489,26 +1417,32 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 				RequestIDFlag,
 			},
 			Action: func(ctx *cli.Context) error {
-				namespaceName := ctx.String(NamespaceFlagName)
+				namespaceName := ctx.String(NamespaceFlag.Name)
 				sinkName := ctx.String(sinkNameFlag.Name)
-				resourceVersion, err := c.getExportSinkResourceVersion(ctx, namespaceName, sinkName)
```

Comment:
are we still going to support this for older clients?

Reply:
You mean getExportSinkResourceVersion? Yes, we do. We do support all operations related to old client. But I really want to deprecate those